state/manager.go

```diff
@@ -42,7 +42,7 @@ func NewManager(endpoint string) *Manager {
 	return &Manager{
 		sinkEndpoint: endpoint,
 		sinkList:     make(map[string]*bus.Sink),
-		sinkChan:     make(chan event, 3),
+		sinkChan:     make(chan event, SinkChanBuffer),
```

Comment:
didn't test this, not sure why it was 3 but I thought that was failing for a large sink count